dask/highlevelgraph.py

```diff
@@ -1207,18 +1207,28 @@ def to_graphviz(
 
     graph_attr = graph_attr or {}
     graph_attr["rankdir"] = rankdir
+    node_attr["shape"] = "box"
     graph_attr.update(kwargs)
+
     g = graphviz.Digraph(
         graph_attr=graph_attr, node_attr=node_attr, edge_attr=edge_attr
     )
 
+    n_tasks_dict = {}
+    for layer in hg.dependencies:
+        n_tasks_dict[layer] = len(hg.layers[layer])
+
+    mn = min(n_tasks_dict.values())
+    mx = max(n_tasks_dict.values())
+
     cache = {}
 
     for k in hg.dependencies:
         k_name = name(k)
         attrs = data_attributes.get(k, {})
+        xfontsize = int(10 + ((n_tasks_dict[k] - mn) / mx) * 20)
```

Comment:
This is a much nicer implementation than the first one you had :smile:

Reply:
Yayyy :rocket:

Reply:
Yayy! :rocket: